### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by tchasinga**
Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
Create a directory for your reusable components: /components.
Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)

### CODE REVIEW

File: src/App.js
Comment (LionRouge1) [inline src/App.js:1]: - [ ] According to the project requirement the router should be set inside the `<app>` component. Therefore I suggest you set your `app` component accordingly to follow the project requirement. You can also read this article to learn how to [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router) in a React component.
Screenshot from 2023-07-20 11-47-00

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @tchasinga, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job setting up the app using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) :1st_place_medal: 
- Nice work adding the `react-router-dom` package :+1: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
- [ ] It is requested in the project requirement that a `components` directory should be created containing a component for individual book, for list of books, for creating new books, and component for the navigation link. Therefore I suggested you to remove all the auto generated file that are no need for this project and create the `components` directory as requested in the project requirement (see image)
Screenshot from 2023-07-20 11-46-48

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        <Routes>
+          <Route
+            path="/"
+            element={<BookList books={books} onDelete={handleDeleteBook} />}
+          />
+          <Route
+            path="/create"
+            element={<BookForm onAddBook={handleAddBook} />}
+          />
```
Comment (codepantha) [inline src/App.js]: - [ ] The `/create` route which you have created is not among the list of routes that are needed for this project. Please create only `/` and `/categories` routes 🙏 
image
- [ ] Also, please create a `<Books />` component which will render your `BooksList` and the `BookForm` component.

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @tchasinga 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Highlights 🎯 
✔️ Descriptive PR
✔️ Added routes with React router dom 📌 
✔️ Linters are passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @tchasinga  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Setup react-router Dom correctly💯
-  Rendered list of book through the right component👍
-  Correct Git flow🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**